crypto.go

```diff
@@ -856,7 +856,7 @@ func (s hkdfScheme) Expand(prk, info []byte, outLen int) []byte {
 }
 
 func (s hkdfScheme) LabeledExtract(salt []byte, suiteID []byte, label string, ikm []byte) []byte {
-	labeledIKM := append([]byte(rfcLabel+" "), suiteID...)
+	labeledIKM := append([]byte(rfcLabel), suiteID...)
```

Comment:
Can you just add the space to the label?

Reply:
If the space isn't always there, that seems like a bug in the spec.

Reply:
We removed the space from the spec (it was previously included in the spec), so everything's aligned now.